app/assets/translations/appConstants/bebbo/UZuz.ts

```diff
@@ -390,10 +391,15 @@ const UZuz = {
               parentGender: "Jins",
               successOnboardingImport: "Farzandingiz haqidagi maʼlumotlatni muvaffaqiyatli import qildingiz",
               updateImportText: "Davom etish uchun zarur maʼlumotlarni yangilang",
-              importOnboardingText: "Farzandingiz maʼlumotlarini oldin eksport qilgan boʻlsangiz, uni import qilishingiz mumkin",
-              OnboardingImportButton: "Hozir import qilish",
+              importOnboardingText: "Bebbo sizning shaxsiy ma'lumotlaringizni yig'maydi.",
+              importOnboardingText1: "Ilovaga kiritgan ma'lumotlaringiz telefoningizda yoki mahaliy xotirangizga saqlanadi.",
```

Comment:
It should be "Ilovaga kiritgan ma'lumotlaringiz telefoningizda yoki shaxsiy bulutli xotirangizda saqlanadi."